src/test/java/resources/batch.feature

```diff
@@ -242,11 +242,25 @@ Feature: Batch Details API test
     Then I should get non empty response
     And I am able to parse batch transactions response
     And I fetch batch ID from batch transaction API's response
-    Then I will sleep for 8000 millisecond
+    Then I will sleep for 10000 millisecond
     And I call the payment batch detail API with expected status of 200
     Then I am able to parse payment batch detail response
     And I should assert total txn count and successful txn count in payment batch detail response
 
+  @gov @batch-teardown
+  Scenario: BD-017 Batch test for de-duplicating payments
+    Given I have the demo csv file "deduplication-test.csv"
+    And I have tenant as "rhino"
+    And I create a new clientCorrelationId
+    And I have private key
+    And I generate signature
+    When I call the batch transactions endpoint with expected status of 202
+    And I am able to parse batch transactions response
+    And I fetch batch ID from batch transaction API's response
+    Then I will sleep for 10000 millisecond
+    And I call the batch details API with expected status of 200
+    Then I should get transactions with note set as "Duplicate transaction"
```

Comment:
add an assert for expected transaction status

Reply:
done